scripts/fetch_and_build_rpm

```diff
@@ -33,7 +33,7 @@ E_O_USAGE
 
 # sets the next version variable used during non-release builds
 setnextversion() {
-    baseversion=$(echo "$1" | tr '~' '-' | cut -d- -f1)
+    baseversion=$(echo "$1" | tr '~' '-' | tr '_' '-' | cut -d- -f1)
```

Comment:
Sample version with beta is 102.4_beta. To be able to get the next version first we need to strip _beta (release stage). This lines strips beta

Reply:
Added comment below
# Below two lines, strips patch version from the version.
    # First line replaces '~' and '_' characters and splits the version by '-' and gets the first
    # from the array.
    # Second line strips '.citus' suffix
    # e.g. input->11.0.0_beta-1.citus; output->11.0.0